File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnFieldAudio.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnAudioFormat;
import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldAudio extends DefnFieldEditable
{
  @Nullable
  public EnumDefnAudioFormat audioFormatType;

  @Nullable
  public Long maxSize;

  @Nullable
  public MetaIdField maxSizeFieldId;

  @Nullable
  public Long maxSizeVar;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldAudio.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnFieldEditable
import com.neome.api.meta.base.Types.EnumDefnAudioFormat
import com.neome.api.meta.base.Types.MetaIdField

interface DefnFieldAudio : DefnFieldEditable
{
  val audioFormatType: EnumDefnAudioFormat?
  val maxSize: Long?
  val maxSizeFieldId: MetaIdField?
  val maxSizeVar: Long?
}